pkg/clusters/addons/kong/kong.go

```diff
@@ -296,3 +360,75 @@ func urlForService(ctx context.Context, cluster clusters.Cluster, nsn types.Name
 
 	return nil, fmt.Errorf("service %s has not yet been provisoned", service.Name)
 }
+
+func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name string) error {
+	license := os.Getenv("KONG_ENTERPRISE_LICENSE")
+	if license == "" {
+		return fmt.Errorf("failed kong enterprise license from environment setting")
+	}
+
+	newSecret := &corev1.Secret{
+		Type: corev1.SecretTypeOpaque,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: map[string][]byte{
+			"license": []byte(license),
+		},
+	}
+
+	_, err := cluster.Client().CoreV1().Secrets(namespace).Create(ctx, newSecret, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("failed creating kong-enterprise-license secret, err %w", err)
+	}
+
+	fmt.Printf("successfully deployed kong-enterprise-license into the cluster .")
+	return nil
+}
+
+func prepareSecrets(ctx context.Context, namespace string) error {
+	stderr := new(bytes.Buffer)
+	pwd := fmt.Sprintf("--from-literal=password=%s", EnterpriseKongAdminDefaultPWD)
+	cmd := exec.CommandContext(ctx, "kubectl", "create", "secret", "generic", EnterpriseAdminPasswordSecretName, "-n", namespace, pwd)
+	cmd.Stdout = io.Discard
+	cmd.Stderr = stderr
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("failed creating super-admin secret %s: %w", stderr.String(), err)
+	}
+	fmt.Printf("successfully created kong admin secret.")
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("failed getting current dir, err %w", err)
+	}
+
+	guiF := pwd + "/secret_conf"
+
+	fi, err := os.Create(guiF)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := fi.Close(); err != nil {
+			fmt.Println("failed closing file %w", err)
+			return
+		}
+	}()
+
+	err = ioutil.WriteFile(guiF, []byte(`{"cookie_name":"04tm34l","secret":"change-this-secret","cookie_secure":false,"storage":"kong"}`), 0600)
+	if err != nil {
+		return fmt.Errorf("failed writing file admin_gui_session_conf, err %v", err)
+	}
+
+	guiFile := fmt.Sprintf("--from-file=admin_gui_session_conf=%s", guiF)
+	portFile := fmt.Sprintf("--from-file=portal_session_conf=%s", guiF)
+	cmd = exec.CommandContext(ctx, "kubectl", "-n", namespace, "create", "secret", "generic", "kong-session-config", guiFile, portFile)
+	cmd.Stdout = io.Discard
+	cmd.Stderr = stderr
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("failed creating kong-session-config secret %s: %w", stderr.String(), err)
+	}
+	fmt.Printf("successfully created kong-session-config secret.")
```

Comment:
we shouldn't print to stdout in library code:

```suggestion
```

Reply:
removed https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R435